### PR TITLE
LLM: add readme for transformer examples

### PR DESCRIPTION
**Pull request by plusbang**
## Description

One part of https://github.com/analytics-zoo/nano/issues/373.
In previous PR (https://github.com/intel-analytics/BigDL/pull/8438), we first fix examples and remove readme temporarily. 
Add readme for two int4 pipelines here.

### 2. User API changes

None

### 4. How to test?
- [x] Unit test
- [x] Local test

### CODE REVIEW

File: python/llm/example/transformers/native_int4_pipeline_readme.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# INT4 Inference Pipeline for Large Language Model using BigDL-LLM Native API
```
Comment (jason-dai) [inline python/llm/example/transformers/native_int4_pipeline_readme.md:1]: BigDL-LLM Native INT4 Inference Pipeline

File: python/llm/example/transformers/native_int4_pipeline_readme.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+In this example, we show a pipeline to convert a large language model to low precision (INT4), and then conduct inference on the converted INT4 model, using BigDL-LLM native API.
```
Comment (jason-dai) [inline python/llm/example/transformers/native_int4_pipeline_readme.md:3]: In this example, we show a pipeline to convert a large language model to bigdl-llm native INT4 format, and then run inference on the converted INT4 model.

File: python/llm/example/transformers/native_int4_pipeline_readme.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+> **Note**: BigDL-LLM native API currently supports model family LLaMA, GPT-NeoX, BLOOM and StarCoder.
```
Comment (jason-dai) [inline python/llm/example/transformers/native_int4_pipeline_readme.md:5]: BigDL-LLM native INT4 format currently supports

File: python/llm/example/transformers/transformers_int4_pipeline_readme.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# INT4 Inference Pipeline for Large Language Model using BigDL-LLM Transformers Style API
```
Comment (jason-dai) [inline python/llm/example/transformers/transformers_int4_pipeline_readme.md:1]: BigDL-LLM Transformers INT4 Inference Pipeline for Large Language Model

File: python/llm/example/transformers/transformers_int4_pipeline_readme.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+In this example, we show a pipeline to convert a large language model to low precision (INT4), and then conduct inference on the converted INT4 model, using BigDL-LLM transformers style API.
```
Comment (jason-dai) [inline python/llm/example/transformers/transformers_int4_pipeline_readme.md:3]: In this example, we show a pipeline to apply BigDL-LLM INT4 optimizations to any Hugging Face Transformers model, and then run inference on the optimized INT4 model.

Comment (plusbang) [reply python/llm/example/transformers/transformers_int4_pipeline_readme.md:3]: > In this example, we show a pipeline to apply BigDL-LLM INT4 optimizations to any Hugging Face Transformers model, and then run inference on the optimized INT4 model.

Done.

Comment (jason-dai) [reply python/llm/example/transformers/transformers_int4_pipeline_readme.md:3]: Not updated?

Comment (plusbang) [reply python/llm/example/transformers/transformers_int4_pipeline_readme.md]: > Not updated?

Sorry for mistake in updating the file before, now it has been updated.

**Review by TheaperDeng — APPROVED** [review]

**Pull request merged**